katas/content/superdense_coding/alice_sends_message/Placeholder.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+
+    operation EncodeMessageInQubit(qAlice : Qubit, message : (Bool, Bool)) : Unit {
+        // Get the bits from the message
+        let (bit1, bit2) = message;
+
+        // Accessing bit1
+        if bit1 {
+            // Implement your solution here...
+
+        }
```

Comment:
Done, I wanted to do it this way since its cleaner.

I dont know if something changed. Earlier, it was complaining about some return type (Bool, Bool) issue and adding if block suppresed that error :-)